expertise/run_test.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+from pathlib import Path
+from tqdm import tqdm
+from collections import OrderedDict, defaultdict
+import multiprocessing
+import pickle
+import copy
+import sys
+
+from .dataset import ArchivesDataset, SubmissionsDataset, BidsDataset
+from .config import ModelConfig
+from .models import bm25
+
+def ranking(archives_dataset, submissions_dataset, publication_id_to_profile_id, worker):
+    # counter = 0
+    for note_id, submission in tqdm(submissions_dataset.items(), total=len(submissions_dataset), position=worker):
+        removed_publication = None
+        for profile_id in publication_id_to_profile_id[note_id]:
+            removed_publication = archives_dataset.remove_publication(note_id, profile_id) or removed_publication
+        bm25Model = bm25.Model(archives_dataset, submissions_dataset, use_title=config['model_params']['use_title'], use_abstract=config['model_params']['use_abstract'])
+        reviewer_scores = bm25Model.score(submission)
+
+        sorted_profile_ids = [(profile_id, value) for profile_id, value in sorted(reviewer_scores.items(), key=lambda item: item[1], reverse=True)]
+        with open(Path('./test/' + note_id + '.pkl'), 'wb') as f:
+            pickle.dump(sorted_profile_ids, f, protocol=pickle.HIGHEST_PROTOCOL)
+        for profile_id in publication_id_to_profile_id[note_id]:
+            archives_dataset.add_publication(removed_publication, profile_id)
+        # counter += 1
```

Comment:
I think this change is good to go. Let's remove the commented code.

Reply:
Done! :)